src/applications/personalization/view-representative/components/ViewRepresentativeDetails.jsx

```diff
@@ -16,35 +44,71 @@ export const ViewRepresentativeDetails = props => (
         {props.telephone}
       </p>
     </div>
-    <h3>What can a representative do?</h3>
+    <h2 className="vads-u-font-size--h3" id="whatCanRepresentativeDo">
```

Comment:
Would it make sense just to make all of these `h2`s that have `vads-u-font-size--h3` `<h3>`s instead, and get rid of the className?

Reply:
Ah, I see, thanks for pointing that out. Disregard then! 👍